resources/mobile/learning-group/session-four.md

```diff
@@ -0,0 +1,363 @@
+# iOS Learning Group Session Three
+
+## Homework review
+
+- What is the purpose of a view controller, from a developer's perspective? What about from a user's perspective?
+
+<details>
+<summary>Answer</summary>
+
+A view controller is the main unit of composition for building native iOS UI's. From a user's perspective, they
+might call it "a screen", since view controllers commonly take up the entire screen of an app. However, view
+controllers really can be _composed_ so while a view controller might indeed take up the entire screen, it might be
+composed of child controllers as the develop sees fit.
```

Comment:
I feel like this is missing some explanation about what a view controller is conceptually responsible for, in MVC.